modules/virt-example-bond-nncp.adoc

```diff
@@ -15,8 +15,6 @@ to the cluster.
 * mode=1 active-backup +
```

Comment:
Article https://access.redhat.com/solutions/67546 mentions that mode 1 is partially working..do I need to call out what might happen on using this mode?